docs/HPCCClientTools/CT_Mods/ECLCC.xml

```diff
@@ -446,6 +446,13 @@
               </row>
 
               <row>
+                <entry>--keywords</entry>
+
+                <entry>Outputs a list of ECL reserved words to stdout (XML
```

Comment:
@JamesDeFabia  Sorry for being pedantic and not catching this before but that should say "Outputs the lists..."